frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/header/header.dart

```diff
@@ -54,7 +52,7 @@ class MenuAppHeader extends StatelessWidget {
           theme: ExpandableThemeData(
             expandIcon: FlowyIconData.drop_down_show,
             collapseIcon: FlowyIconData.drop_down_hide,
-            iconColor: theme.shader1,
```

Comment:
The colors in AppTheme are specified by the design specs. I think keeping those names is good for development.